TorchSharp/NativeMemoryManager.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Buffers;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System;
+
+namespace TorchSharp
+{
+    public class NativeMemory<T> : MemoryManager<T>
```

Comment:
`public sealed`?

Reply:
I copied and pasted the native thing from SNT/Test but yes make sense to have it `sealed`.